README.md

```diff
@@ -1 +1,127 @@
-# Book-store
\ No newline at end of file
+# BOOK STORE
+
+
+
+A book store website displays a list of books, add new book and remove selected book.
+
+# 🛠 Built With
+### Tech Stack:
+- HTML. 
+- CSS.
+- Javascript ES6.
+- React
+
+### Server:
+- local.
+### Database:
+- N/A.
+
+# Key Features:
+- The project uses React and Redux
+- CSS3.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here you can see the live demo of my project:
+
+Currently not available
+
+<!-- - [Live Demo Link](https://johnie261.github.io/to-do-list/dist/) -->
+
+# 💻 Getting Started:
+### To get a local copy up and running, follow these steps:
+
+# Prerequisites:
+### In order to run this project you need:
+
+ - A PC with window, linux and mac OS operating systems.
+ 
+ - Installed Git.
+
+ - Have knowledge of Webpack, Git and GitHub.
+
+ - An internet connection 
+
+ - A browser such as chrome, Firefox and Internet explorer.
+
+ - Have a code editor
+
+# Setup:
+Clone this repository to your desired folder:
+
+
+- Clone this repository to your desired folder.
+- Run: git clone + 'link copied' and make sure the file is copied.
+- Open the file with a code edi
+
+# Install:
+### Install this project with:
+- Git.
+- Github.
+
+# Usage:
+### To run the project, execute the following command:
+To run the project, execute the following command:
+
+
+- To run the project you need to make use of git commands such as git add , git pull and git command
+
+# Run tests:
+### To run tests, run the following command:
+ 
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint "**/*.{js,jsx}"
+```
+To run the project:
+
+```
+
+npm start
+```
+
+# Deployment:
+### You can deploy this project using:
+- Github pages.
+
+# 👥 Authors:
+### 👤 Johnie:
+- GitHub: [github](https://github.com/johnie261)
+- Twitter: [twitter](https://twitter.com/njorogejohnie)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/john-njoroge-19b6a4245/)
+
+# 🔭 Future Features:
+- Use Redux in React components.
+- Add reducers and actions.
+- Fetching books from API.
+
+
+# 🤝 Contributing:
+Contributions, issues, and feature requests are welcome!
+Feel free to check the issues page.
+
+# ⭐️ Show your support
+If you like this project kindly send to me an e-mail expressing it, it would make my day and fuel my motivation.
+
+# 🙏 Acknowledgments:
+I would like to thank Microverse team for this journey.
+
+Original design by [Zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed
```

Comment:
- [ ] It would be nice if your readme file would be consistent with the microverse readme template 👍 because there is some problem in your readme which are needed to be addressed 😊

1. Some headings are not of the size which is required in the microverse readme template
2. Some sections have no data and they are empty. Kindly remove those sections
3. You are missing the Table of contents in your readme file 🤔 Kindly add it in your readme

Reply:
Thank @HAMMAS-SALEEM , let me make the changes